src/wazuh_modules/wm_vulnerability_scanner.c

```diff
@@ -132,5 +132,9 @@ cJSON* wm_vulnerability_scanner_dump(wm_vulnerability_scanner_t * data)
 {
     cJSON *root = cJSON_CreateObject();
     cJSON_AddItemToObject(root, "vulnerability-detection", cJSON_Duplicate(data->vulnerability_detection, TRUE));
+    cJSON_DeleteItemFromObject(cJSON_GetObjectItem(root, "vulnerability-detection"), "index-status");
```

Comment:
```suggestion
    cJSON_DeleteItemFromObject(cJSON_GetObjectItem(root, "vulnerability-detection"), "indexer-status");
```

Reply:
Thanks!

Reply:
@Dwordcito FYI, the `cJSON_DeleteItemFromObject` function doesn't fail if the object received is null:

```C++
CJSON_PUBLIC(void) cJSON_DeleteItemFromObject(cJSON *object, const char *string)
{
    cJSON_Delete(cJSON_DetachItemFromObject(object, string));
}
```
> [!NOTE]
> cJSON_DetachItemFromObject returns null if the object doesn't exist

```C++
CJSON_PUBLIC(void) cJSON_Delete(cJSON *item)
{
    cJSON *next = NULL;
    while (item != NULL)
    {
        next = item->next;
        if (!(item->type & cJSON_IsReference) && (item->child != NULL))
        {
            cJSON_Delete(item->child);
        }
        if (!(item->type & cJSON_IsReference) && (item->valuestring != NULL))
        {
            global_hooks.deallocate(item->valuestring);
        }
        if (!(item->type & cJSON_StringIsConst) && (item->string != NULL))
        {
            global_hooks.deallocate(item->string);
        }
        global_hooks.deallocate(item);
        item = next;
    }
}
```

> [!NOTE]
> The function returns safely if the item passed is null